pkg/tsdb/elasticsearch/response_parser_test.go

```diff
@@ -691,17 +691,17 @@ func TestResponseParser(t *testing.T) {
 			frame := dataframes[0]
 			require.Len(t, frame.Fields, 2)
 			require.Equal(t, frame.Fields[0].Name, "time")
-			require.Equal(t, frame.Fields[0].Len(), 2)
```

Comment:
please also adjust the test-data at https://github.com/jorgelbg/grafana/blob/es-alert-trimedges/pkg/tsdb/elasticsearch/response_parser_test.go#L647 so that the `trimEdges` value is a string.

Reply:
this is not needed anymore, now we need to have two tests, one for string and one for number.